File: test_archives/py_functions/batch2/function9.py
def function9(int1, float1, str1, list1):
  # int1 is integer 
  # float1 is float
  # str1 is string
  # list1 is list
  
  total = int1 + float1 + len(str1)
  
  for i in list1:
    total += i
  
  return total

File: test_archives/cpp_functions/batch3/function9.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

string function9(string string, int int1, int int2, int int3, vector<int> list1) {
  int total = 0;
  for (int i : list1) {
    total += (i + int1 + int2 + int3);
  }
  
  return string + to_string(total);
}